src/modules/blockchain/implementation/web3-service.js

```diff
@@ -573,12 +573,12 @@ class Web3Service {
         );
 
         return {
-            startTime: result['0'],
```

Comment:
I think we should be consistent with how we process contract responses, we either should convert everything in Web3 Service or do it in other places.